packaging/osx/sharedhost/scripts/postinstall

```diff
@@ -11,6 +11,8 @@ INSTALL_DESTINATION=$2
 chmod 755 $INSTALL_DESTINATION/dotnet
 
 # Add the installation directory to the system-wide paths
+# But first create the directory if it doesn't exist
+mkdir -p /etc/paths.d
```

Comment:
Confirmed it works. On fresh install the `/etc/paths.d` does not exist. After running the PKG with this change, the folder and the `dotnet` file in it got created and the host was on the path after a successful install.